src/models/NavItemPage.php

```diff
@@ -472,6 +483,12 @@ public static function getBlockItem(NavItemPageBlockItem $blockItem, NavItemPage
         $blockObject->setVarValues((empty($blockValue)) ? [] : $blockValue);
         $blockObject->setCfgValues((empty($blockCfgValue)) ? [] : $blockCfgValue);
 
+        if (is_array($envOptions)) {
```

Comment:
i would rather recommend to use array $envOptions = [], instead of mixing arrays and nullable values.

Reply:
@nadar has changed it in https://github.com/luyadev/luya-module-cms/pull/403